tools/pd-ctl/pdctl/command/store_command.go

```diff
@@ -243,18 +244,50 @@ func storeLimitSceneCommandFunc(cmd *cobra.Command, args []string) {
 	}
 }
 
+func convertToStoreInfo(content string) string {
+	store := &api.StoreInfo{}
+	err := json.Unmarshal([]byte(content), store)
+	if err != nil {
+		return content
+	}
+	store.Store.State = 0
+	byteArr, err := json.MarshalIndent(store, "", "  ")
+	if err != nil {
+		return content
+	}
+	return string(byteArr)
+}
+
+func convertToStoresInfo(content string) string {
+	stores := &api.StoresInfo{}
+	err := json.Unmarshal([]byte(content), stores)
+	if err != nil {
+		return content
+	}
+	for _, store := range stores.Stores {
```

Comment:
What about adding a comment here to explain why we change this to 0?

Reply:
done